### DISCUSSION TITLE
Rotary servo features? (with Balormk device)

### DISCUSSION THREAD

**Discussion by plaes** [meerk40t/meerk40t#2476] [Q&A] [answered]
Heya!

Trying to wrap my head around the "Rotary" feature in current 0.9.4001 (git) release.

I'm using the `balormk` device, but I cannot find the UI for configuring servo parameters and there doesn't seem to be any UI elements to spin the rotary attachment.

**Comment by jpirnay** [comment, marked as answer]
Unfortunately there is no support as of now for an external rotary. We will work on this but there is no ETA
